examples/apps/api-gateway-multiple-origin-cors/package-lock.json

```diff
@@ -3339,9 +3339,9 @@
       "dev": true
     },
     "mixin-deep": {
-      "version": "1.3.1",
-      "resolved": "https://registry.npmjs.org/mixin-deep/-/mixin-deep-1.3.1.tgz",
-      "integrity": "sha512-8ZItLHeEgaqEvd5lYBXfm4EZSFCX29Jb9K+lAHhDKzReKBQKj3R+7NOF6tjqYi9t4oI8VUfaWITJQm86wnXGNQ==",
+      "version": "1.3.2",
```

Comment:
@michaeljfazio Could you please update this PR by removing this commit from this PR

Reply:
Rebased back off the release tag now to avoid picking up the change that introduced it. Should be good to go now.